git-extra/getgit

```diff
@@ -188,72 +191,79 @@ process "/mingw$bit/share/git-core"
 process "/mingw$bit/share/git-gui"
 process -f "/mingw$bit/share/perl5"
 process -f "/mingw$bit/ssl"
-process "/cmd"
-process -f "/etc/profile.d"
+if [[ ! x$no_env == x1 ]]; then
+    process "/cmd"
+    process -f "/etc/profile.d"
+else
+    cp -f $gitupk/cmd/* $bin
```

Comment:
Does it make sense to copy these files to `/usr/bin` or `/bin`? Wouldn't they break MSYS2/Cygwin `git`? And wouldn't they be broken in `/usr/bin` anyways, because they expect `../mingw64/bin/git.exe` etc to exist?

Reply:
Most files in `/cmd` are compiled from [`git-wrapper.c`](https://github.com/git-for-windows/MINGW-packages/blob/main/mingw-w64-git/git-wrapper.c) and can't rely on `PATH` to define `PATH`.

Reply:
Ah. Would the PR be acceptable if /cmd is left as is (and not copied to somewhere else) i.e. only skips /etc/profile.d, the registry PATH setting, and the context menu scripts? I can add a message informing the user they need to add /cmd to path themselves.